trollmoves/dispatcher.py

```diff
@@ -434,15 +474,20 @@ def dispatch(source, destinations):
         message = "Source file for dispatching does not exist:{}".format(str(source))
         logger.error(message)
         any_error = True
+    success = {}
     # rename and send file with right protocol
-    for url, params in destinations:
+    for url, params, client in destinations:
         try:
             logger.debug("Dispatching %s to %s", source, str(clean_url(url)))
             move_it(source, url, params)
+            success[client] = True
```

Comment:
I thinks bad things are happening if we have multiple destinations on the same client

Reply:
This should be tested

Reply:
Oh, that's possible? Ok, will have to think about it a bit.

Reply:
As discussed on Slack, I added an exception here in the case there were multiple clients with the same name. And added few tests for `dispatch()`.